src/leiningen/jlink.clj

```diff
@@ -113,8 +178,5 @@
    (cond
      (= sub "init") (init project)
      (= sub "clean") (clean project)
-     (= sub "run") (run project args)
-     (= sub "test") (test project args)
      (= sub "assemble") (assemble project)
-     (= sub "package") (package project)
```

Comment:
Our original `package` command was to create a tarball of both jre and application code for distribution. I suggest to keep this function.

Reply:
Okay, I can look into this. The code we had in place would shell out to call `tar` and that doesn't work for me (I am on Windows 😝 ). I bet there's some code in Apache Commons that will do much the same thing, though.

Reply:
I see. Just in a quick search I find [lein-tar](https://github.com/technomancy/lein-tar) that can be modified to fit our workflow. I think we can leave this feature to lein-tar and I will seek to create a PR for it to package a custom target path.